opencti-platform/opencti-front/src/private/components/data/sync/SyncEdition.jsx

```diff
@@ -208,15 +214,39 @@ const SyncEditionContainer = ({ synchronizer }) => {
               style={{ marginTop: 20 }}
               disabled={true}
             />
-            <Field
-              component={TextField}
-              variant="standard"
-              name="token"
-              label={t_i18n('Remote OpenCTI token')}
-              fullWidth={true}
-              style={{ marginTop: 20 }}
-              disabled={true}
-            />
+            <div
```

Comment:
Why not using the new field dedicated to show/hide secrets?

Reply:
And I think the field should be named `PasswordTextField` instead of `ToggleVisibilityField` because it is not a toggle. It's a text field where you can show/hide the content